salt/modules/glanceng.py

```diff
@@ -0,0 +1,205 @@
+# -*- coding: utf-8 -*-
+'''
+Glance module for interacting with OpenStack Glance
+
+.. versionadded:: Nitrogen
+
+:depends:shade
+
+Example configuration
+
+.. code-block:: yaml
+    glance:
+      cloud: default
+
+.. code-block:: yaml
+    glance:
+      auth:
+        username: admin
+        password: password123
+        user_domain_name: mydomain
+        project_name: myproject
+        project_domain_name: myproject
+        auth_url: https://example.org:5000/v3
+      identity_api_version: 3
+'''
+
+from __future__ import absolute_import
+
+import salt.utils
```

Comment:
The `salt.utils` functions have been deprecated and moved to new files recently. We're moving away from importing `salt.utils` directly. Please use the new paths for any utils functions you may need.

Reply:
They are using salt.utils.clean_kwargs for backwards compatibility with this

Reply:
Backwards compatibility with what? This PR is being opened against develop.

Reply:
some people will want to drop this into their already functioning 2017.7 setup.